### PR TITLE
added ssl configs for frontend, consumer and management

### PR DESCRIPTION
**Pull request by spattanaik75**

### CODE REVIEW

File: hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java [hunk 1]
```diff
@@ -88,9 +88,20 @@
 
     KAFKA_AUTHORIZATION_ENABLED("kafka.authorization.enabled", false),
     KAFKA_AUTHORIZATION_MECHANISM("kafka.authorization.mechanism", "PLAIN"),
+    //SASL AUTHENTICATION
     KAFKA_AUTHORIZATION_PROTOCOL("kafka.authorization.protocol", "SASL_PLAINTEXT"),
     KAFKA_AUTHORIZATION_USERNAME("kafka.authorization.username", "username"),
     KAFKA_AUTHORIZATION_PASSWORD("kafka.authorization.password", "password"),
+    //SSL AUTHENTICATION
+    //KAFKA_AUTHORIZATION_PROTOCOL("kafka.authorization.mechanism", "SSL"),
```
Comment (piotrrzysko) [inline hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java:96]: Is there any reason to leave this line commented out? Btw, the enum name is inconsistent with the property name.

File: docker/latest/consumers/consumers.properties [hunk 2]
```diff
@@ -12,3 +12,13 @@ consumer.workload.consumers.per.subscription=1
 
 schema.cache.refresh.after.write.minutes=1
 schema.repository.serverUrl=http://schema-registry:8081
+
+#kafka.authorizaiton.enabled=true
```
Comment (piotrrzysko) [inline docker/latest/consumers/consumers.properties:16]: Until the keystore and truststore are added to the images in my opinion, it is unnecessary to leave these properties commented out.

**Review by piotrrzysko — CHANGES_REQUESTED** [review]
Tests are failing: 
```
pl.allegro.tech.hermes.integration.AttachingKeepAliveHeaderTest.setup FAILED
    org.junit.ComparisonFailure: expected:<[201]> but was:<[500]>
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at pl.allegro.tech.hermes.test.helper.endpoint.HermesAPIOperations.createTopic(HermesAPIOperations.java:69)
        at pl.allegro.tech.hermes.test.helper.endpoint.HermesAPIOperations.createTopic(HermesAPIOperations.java:62)
        at pl.allegro.tech.hermes.test.helper.endpoint.HermesAPIOperations.createTopic(HermesAPIOperations.java:58)
        at pl.allegro.tech.hermes.test.helper.endpoint.HermesAPIOperations.createTopic(HermesAPIOperations.java:54)
        at pl.allegro.tech.hermes.test.helper.endpoint.HermesAPIOperations.buildTopic(HermesAPIOperations.java:128)
        at pl.allegro.tech.hermes.integration.AttachingKeepAliveHeaderTest.setup(AttachingKeepAliveHeaderTest.java:27)
```

It seems that it is caused by a changed default value of `pl.allegro.tech.hermes.management.config.kafka.KafkaProperties#bootstrapKafkaServer` - `hermes-management` is unable to connect to Kafka.

**Comment by spattanaik75** [comment]
@piotrrzysko apologies, i have been caught up with office works lately. will take a look at it when i have a chance.

File: hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java (hunk 1 above)
Comment (spattanaik75) [reply hermes-common/src/main/java/pl/allegro/tech/hermes/common/config/Configs.java:96]: you don't have to use this line at all, the property is read from the management.yaml file.

**Comment by Diewa** [comment]
Closing due to no activity. Feel free to re-open if needed.

**Pull request closed without merging**